### PR TITLE
More Durable Objects doc improvements

### PR DESCRIPTION
**Pull request by a-robinson**

### CODE REVIEW

File: products/workers/src/content/learning/using-durable-objects.md [hunk 1]
```diff
@@ -154,186 +153,153 @@ export class Counter {
...
-    }
+Note that in the example below, we have written the fetch handler using a new kind of Workers syntax based on ES modules. We have done this because Durable Objects must be implemented in Workers that use this new syntax, and by implementing our fetch handler in this new syntax we can put it in the same Worker as the Durable Object and deploy them as a single unit. We recommend following this approach of implementing Durable Objects and a corresponding fetch handler in the same Worker not only because it is convenient, but also because as of today it is not possible to upload a Worker that does not implement a fetch handler.
 
-}
+Note that in the example below, we have written the fetch handler using a new kind of Workers syntax based on ES modules. This syntax is required for Durable Objects. The fetch handler in this example implements the Worker that talks to the Durable Object. We recommend following this approach of implementing Durable Objects and a corresponding fetch handler in the same script not only because it is convenient, but also because as of today it is not possible to upload a script to the runtime that does not implement a fetch handler.
```
Comment (koeninger) [inline products/workers/src/content/learning/using-durable-objects.md:170]: I think this was intended to replace the paragraph above?

Comment (a-robinson) [reply products/workers/src/content/learning/using-durable-objects.md:170]: Heh, thanks for checking this more closely than I clearly did.

File: products/workers/src/content/learning/using-durable-objects.md [hunk 2]
```diff
@@ -154,186 +153,151 @@ export class Counter {
...
+    let stub = await env.EXAMPLE_CLASS.get(id);
+
+    // Forward the request to the Durable Object. Note that `stub.fetch()` has
+    // the same signature as the global `fetch()` function, except that the
+    // request is always sent to the object, regardless of the request's URL.
+    //
+    // The first time we send a request to a new object, the object will be
+    // created for us. If we don't store durable state in the object, it will
+    // automatically be deleted later (and recreated if we request it again).
+    // If we do store durable state, then the object becomes permanent.
```
Comment (koeninger) [inline products/workers/src/content/learning/using-durable-objects.md]: Minor, but given the preceding sentence I could see someone misreading "object becomes permanent" as "object won't be evicted and recreated"

**Review by koeninger — APPROVED** [review]

File: products/workers/src/content/learning/using-durable-objects.md (hunk 2 above)
Comment (a-robinson) [reply products/workers/src/content/learning/using-durable-objects.md]: Sure, clarified a bit.

**Pull request merged**